site/src/routes/package/[name]/components/primitive-name-tooltip.tsx

```diff
@@ -1,16 +1,54 @@
 import { createResizeObserver } from "@solid-primitives/resize-observer";
 import { createMutationObserver } from "@solid-primitives/mutation-observer";
-import { batch, Component, createRoot, createSignal, Match, onMount, Switch } from "solid-js";
+import { Accessor, batch, Component, createRoot, createSignal, JSX, onMount } from "solid-js";
 import { useTippy } from "solid-tippy";
 import { Content } from "tippy.js";
 import { BASE } from "~/constants";
 import { BundlesizeItem } from "~/types";
 
-function createTooltipContent(
-  el: HTMLElement,
-  data: BundlesizeItem,
-  type: ReturnType<typeof getTypeOfPrimitive>,
-): Content {
+export type PrimitiveType = "create" | "use" | "make" | "get" | "component" | "utility";
```

Comment:
Love it!!

> That's fair. I wanted to simplify it, as there only point of this naming choice is to separate reactive primitives (`create___` & `use___`), from your usual, vanilla-js ones (`make___`, `get___`, etc.). And this is the distinction worth emphasizing.
> But I agree that after the change, the tooltip became a little less helpful — it could tell a bit more about the meaning behind that specific prefix, or link to the more detailed explanation:
> - `get___` - pure function calculating the result based on the params (implicit and explicit ones).
> - `make___` - function instantiating a subject from the name (can have side-effects)
> - `create___` - a function creating a new instance of a reactive subject (can hook into existing reactivity graph for scheduling effects or disposal)
> - `use___` - same as `create___` but it can reuse or connect to an existing reactive subject through context or other means (placement in the reactive graph matters more than with `create___`)
> - \+ a component

It would be cool to see these longer explanations, with least amount of text since tooltips should be small and simple IMO, but I'm good with the current changes 👍